src/color.scss

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2021 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/* stylelint-disable custom-property-empty-line-before, declaration-no-important */
+
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+:host {
+  background-color: var(--ds-auro-menuoption-background-color);
+  color: var(--ds-auro-menuoption-text-color);
+
+  ::slotted(hr) {
+    border-top-color: var(--ds-auro-menu-border-color) !important;
+  }
```

Comment:
Shouldn't these 3 lines be going into `auro-menu` and not `auro-menuoption`?

Reply:
Good catch